### ISSUE TITLE
Better Content Warnings

### ISSUE THREAD

**Issue by neonpixii** [pixelfed/ideas#28]
As it stands the content warning system is not particularly effective; there's no field for entering in what the sensitive content is. You just get a generic message. 

Being able to change the message from "Content Warning: This may contain potentially sensitive content" to something more descriptive for each individual post would be a huge benefit for usablitiy.

Additionally, displaying the custom content warning text on the greyed-out thumbnail in the profile view too would help people find a content-warned image in a profile that has had heavy use better (though maybe that would be best put into a separate issue)

**Comment by spaceottercode** [comment]
I believe the first request of the OP is now implemented. Not sure if the second point remains applicable. The timeline now supports the display of custom CW, but these are not shown in the profile view. it is unclear to me if the OP meant the timeline or profile view. If the former, then this issue may be resolved...

**Comment by flypenguin** [comment]
yes, really, _please please please_ implement the option "disable sensitive content warning". I know what i'm doing, i'm not offended and i am clearly able to block or unfollow, and i consider this _incredibly_ annoying. also some people simply flag a naked shoulder as "sensitive" – i do not share this point of view.
